saleor/discount/migrations/0068_merge_20231106_1013.py

```diff
@@ -0,0 +1,12 @@
+# Generated by Django 3.2.22 on 2023-11-06 10:13
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("discount", "0064_alter_promotionrule_catalogue_predicate"),
+        ("discount", "0067_voucher_single_use"),
+    ]
+
+    operations = []
```

Comment:
Do we need an empty migration?

Reply:
This is a merge of migrations, since the changes need to be ported to 3.17 version